lib/JsModule.js

```diff
@@ -201,7 +201,10 @@ export default class JsModule {
       defaultImport,
       importFunction: config.get('importFunction', { pathToImportedModule }),
       namedImports,
-      path: this.importPath,
+      path: config.get('moduleSpecifier', {
+        pathToImportedModule,
+        moduleSpecifier: this.importPath,
+      }),
```

Comment:
Is it OK that the "path" specified to ImportStatement has been changed from one that might have been useful for actually finding a file to one that is only useful as a ModuleSpecifier in an import statement? If so, should it be called "path"?